core/src/main/java/com/hcl/ecomm/core/config/ProductServiceConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.hcl.ecomm.core.config;
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.AttributeType;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "Product Service Configuration")
```

Comment:
@nitheshkumar-hcl : Name of the service should be Magento Product Service Configuration

Reply:
product service changed to agento Product Service Configuration